scri/SpEC/file_io/__init__.py

```diff
@@ -429,18 +468,30 @@ def create_abd_from_h5(file_format, convention="SpEC", **kwargs):
 
     Parameters
     ----------
-    file_format : 'SXS', 'CCE', 'SpECTRECCE1', 'RPDMB', or 'RPXMB'
+    file_name : str,
+        Path to .h5 file to be loaded.
+    file_format : 'SXS', 'SpECTRECCE', 'RPDMB', or 'RPXMB'
```

Comment:
Either's fine with me. I would say that the current format can reasonably be considered the first public version. Having said that, keeping the number the same between this and what we have inside the CCE output files could be helpful.

Reply:
I was hoping we could just have the user specify `SpECTRECCE` and then the code could figure out how to extract data from the output file based on what version it is. The way I've set the code setup works for the previous version of SpECTRE CCE as well as the new version, and I figure we can just change it to work for whatever future versions there may be. Thoughts?

Reply:
That's reasonable, but to ensure that doesn't fail in the future, we'd need to check that the input data actually has an `sxs_format = "SpECTRECCE_v1"` attribute, because we won't want this function to silently fail on `v2`.  (Then, to add `v2` support, we'll need to add new code anyway.)